extensions/markdown-language-features/src/markdownEngine.ts

```diff
@@ -246,6 +246,7 @@ export class MarkdownEngine {
 								scheme: uri.scheme,
 								fragment: uri.fragment,
 								query: uri.query,
+								path: '/file' + fileUri.path
```

Comment:
Can you explain what this change does?

It don't doesn't seem correct to me. Even if it works in this case, I suspect it won't work on non-local file systems